components/script/dom/htmlmediaelement.rs

```diff
@@ -1426,6 +1446,13 @@ impl HTMLMediaElementMethods for HTMLMediaElement {
         USVString(self.current_src.borrow().clone())
     }
 
+    // https://html.spec.whatwg.org/multipage/#attr-media-crossorigin
+    fn CrossOrigin(&self) -> DOMString {
+        DOMString::from_string(self.cross_origin.borrow().clone())
+    }
+    // https://html.spec.whatwg.org/multipage/#attr-media-crossorigin
+    make_setter!(SetCrossOrigin, "crossorigin");
```

Comment:
This won't work as written, since the cross_origin member is never updated with the new value. We should do what HTMLImageElement does instead:
```rust
    // https://html.spec.whatwg.org/multipage/#attr-media-crossOrigin
    fn GetCrossOrigin(&self) -> Option<DOMString> {
        reflect_cross_origin_attribute(self.upcast::<Element>())
    }

    // https://html.spec.whatwg.org/multipage/#attr-media-crossOrigin
    fn SetCrossOrigin(&self, value: Option<DOMString>) {
        set_cross_origin_attribute(self.upcast::<Element>(), value);
    }
```